docs/ops/activation/HSigmoid_5.md

```diff
@@ -0,0 +1,47 @@
+## HSigmoid<a name="HSigmoid"></a> {#openvino_docs_ops_activation_HSigmoid_5}
+
+**Versioned name**: *HSigmoid-5*
+
+**Category**: *Activation function*
+
+**Short description**: HSigmoid takes one input tensor and produces output tensor where the hard version of sigmoid function is applied to the tensor elementwise.
+
+**Detailed description**: For each element from the input tensor calculates corresponding
+element in the output tensor with the following formula: 
+
+    \f[
+    HSigmoid(x) = \frac{min(max(x + 3, 0), 6)}{6}
+    \f]
+
+The HSigmoid operation is introduced in the following [article](https://arxiv.org/pdf/1905.02244.pdf).
+
+**Attributes**: operations has no attributes.
+
+**Inputs**:
+
+* **1**: An tensor of type T. **Required.**
```

Comment:
Please add two new inputs for `alpha` and `beta` so this way we specify `HardSigmoid` (y = max(0, min(1, alpha * x + beta))) as in ONNX, MxNet, etc. and make sure that it will be aligned with TF Keras HardSigmoid and one in a paper for MobileNet V3

Reply:
But actually we already have `HardSigmoid` like ONNX in the current OpenVino opset [HardSigmoid_1.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/HardSigmoid_1.md) this is why I wanted to understand the difference, because I know that `H` letter in `HSigmoid` also stands for Hard. 
Now after quick research, I see that for MobileNet it's optimized with Relu6 so I think it's ok to introduce `HSigmoid` as a new operator.

Reply:
Agree, let us proceed with `HSigmoid`